CHANGELOG.md

```diff
@@ -4,6 +4,51 @@
 
 ### Notable changes
 
+* **V8**: upgrade to 4.2.77.18, see the [ChangeLog](https://chromium.googlesource.com/v8/v8/+/refs/heads/4.2.77/ChangeLog) for full details. Notable items:
+  - Classes have moved out of staging, the `class` keyword is now usable without flags
```

Comment:
s/,/;

Also want to mention object literal extensions (concise methods and property shorthands).

Maybe mention that like let and const, class is only usable in strict mode.